lib/trino-record-decoder/src/main/java/io/trino/decoder/avro/AvroColumnDecoder.java

```diff
@@ -185,6 +192,12 @@ public boolean getBoolean()
         public long getLong()
         {
             if (value instanceof Long || value instanceof Integer) {
+                if (columnType == TimestampType.TIMESTAMP_MILLIS) {
+                    return ((Number) value).longValue() * Timestamps.MICROSECONDS_PER_MILLISECOND;
+                }
+                else if (columnType == TimeType.TIME_MILLIS) {
```

Comment:
redundant `else` here

Reply:
I think it's a style issue. If Trino's style prevents it, I can remove it. Otherwise I'd like to keep it as is.

Reply:
We avoid redundant `else`. 
https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#additional-ide-configuration

Reply:
OK. Updating...

Reply:
```suggestion
                if (columnType == TimeType.TIME_MILLIS) {
```